scripts/inc.Makefile

```diff
@@ -0,0 +1,53 @@
+RM         ?= rm
+CP         ?= cp
+MV         ?= mv
+
+GO         ?= go
+GOFMT      ?= gofmt
+GOLINT     ?= golint
+GOTEST     ?= go test
+GOTOOL     ?= go tool
+
+DOCKER     ?= docker
+
+LDFLAGS     =
+
+all: build
+
+help: ## Display this help
+	@awk 'BEGIN {FS = ":.*?## "; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z0-9._-]+:.*?## / {printf "  \033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
```

Comment:
Not sure if I understand how this one-liner works, but I just ran `make help` locally and wow, nice!

Reply:
credits to @odacremolbap